### PR TITLE
dev: Add Keystone embed with Bookmark and Collection

### LINKED ISSUE #78: Set up embedded Keystone with a dummy model
Set up embedded Keystone with a dummy model (could be Post, Bookmark, etc.)

### PR DESCRIPTION
**Pull request by suzubara**
## Description 

This PR adds [Keystone CMS in embedded mode](https://keystonejs.com/docs/walkthroughs/embedded-mode-with-sqlite-nextjs#bonus-add-the-graph-ql-api-to-the-frontend) so that we can develop the portal application against a CMS, prior to standing the CMS up as its own service. This approach is described in [this ADR](https://github.com/USSF-ORBIT/ussf-portal/blob/main/docs/adr/0006-use-keystone-cms.md).

The Keystone entrypoint is added at [`keystone.ts`](https://github.com/USSF-ORBIT/ussf-portal-client/pull/213/files#diff-f1e82ef6c3a24515123a6ff6f1dfc3e0464caf29be7db0fe0bd9279521d1413b) -- this is where Keystone bootstraps all config, database settings, and the content schema. I added a basic schema for Bookmarks and Collections, as well as some corresponding tests.

`schema.graphql` and `schema.prisma` are both generated by Keystone but should be checked into VCS.

This also exposes a local URL for viewing the GraphQL API at: http://localhost:3000/api/graphql

I also did a little bit of housekeeping, updating `.eslintignore` & `.prettierignore` files.

Fixes #78 

## Review Notes

This has to be tested locally since Keystone-embed only works in local development. To test, check out this branch and run:
```
yarn
yarn dev
```

This should expose two URLs:
http://localhost:3000/ - the portal site
http://localhost:8000/ - the Keystone admin UI

Verify you can add Bookmarks/Collections as expected. Note that these aren't surfaced in the UI anywhere yet, so there is nothing to test in the portal itself (yet!). If you add content using the admin UI, there should be changes made to the `cms.db` file.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,11 +84,14 @@
...
     "standard-version": "9.3.1",
-    "storybook-addon-next-router": "3.0.7",
+    "storybook-addon-next-router": "3.0.8",
     "storybook-css-modules-preset": "1.1.1",
     "style-loader": "2.0.0",
     "typescript": "4.4.2"
   },
+  "resolutions": {
+    "@babel/runtime": "7.15.4"
+  },
```
Comment (suzubara) [inline package.json:94]: I ended up updating a bunch of packages because there were some incompatibilities causing Storybook to not run.

**Review by abbyoung — APPROVED** [review]
awesome!

File: jest.config.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
 module.exports = {
   testPathIgnorePatterns: ['/node_modules/', '.next/', 'cypress/'],
-  moduleDirectories: ['node_modules/', '.'],
```
[the remark this answers is not in the document]
Comment (abbyoung) [reply jest.config.js:3]: what does this change (if anything) moving forward?

[the remark this answers is not in the document]
Comment (suzubara) [reply jest.config.js:3]: If we add directories under `src/` that we want to be able to import from using absolute paths, they'll need to be added to this file so Jest can resolve the imports correctly

**Pull request merged**